Content.Server/Zombies/ZombieSystem.cs

```diff
@@ -254,7 +254,7 @@ private void OnMeleeHit(EntityUid uid, ZombieComponent component, MeleeHitEvent
 
                 if (HasComp<ZombieComponent>(entity))
                 {
-                    args.BonusDamage = -args.BaseDamage * zombieComp.OtherZombieDamageCoefficient;
+                    args.BonusDamage = -args.BaseDamage;
```

Comment:
Based on your other comments, I'm not sure if this is doing what you meant to do.  Previously, zombie friendly fire damage was limited by the coefficient (see how it multiplies it by .25, basically making it quarter damage)... with this change, it seems like zombie vs zombie would do full damage again.

Reply:
Nope, it does damage - damage = 0, it's the same formula used on infected zombies in the same class.

Reply:
Oh, huh, interesting.  Thanks for the explanation